_sass/method.scss

```diff
@@ -221,25 +221,8 @@ display:table;margin-top: 3rem;}
 
 }
 
-
-
-.category--discover .method--title {
-  border-color: $color-blue;
-}
-
-
-.category--decide .method--title {
-  border-color: $color-green;
-}
-
-.category--make .method--title {
-  border-color: $color-gold;
-}
-
-.category--validate .method--title {
-  border-color: $color-red;
-}
-
-.category--fundamentals .method--title {
-  border-color: $color-blue-dark;
-}
+.category--discover .method--title {      border-color: $color-blue; }
```

Comment:
I'd recommend following the css formatting guidelines here https://frontend.18f.gov/#css-formatting